README.md

```diff
@@ -73,6 +73,14 @@ Line Mode command; the selection is moved linewise.)
 - `.moveAndRenameFile` or `:Move`: Move and Rename the current file. Keeps the
 old name if the new path ends with `/`. Works like the UNIX `mv` command. Best
 used with [autocompletion of directories](#autocompletion-of-directories).
+- `.trashFile`: Trash the current file.
+	- Use `trashCmd` to specify an external trash command. It defaults to
+`gio trash` on *Linux*, `trash` on *Mac* and *Windows*.
+	- Otherwise specify `trashLocation` to move the file to that directory. It
```

Comment:
I think it's relevant to add a sentence that when `trashCmd` and `trashLocation` are both specified, `trashLocation` is ignored. Users might assume that setting both specific where the `trashCmd` moves the file to, which is not the case.